profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/ConfirmedFriends.java

```diff
@@ -133,24 +133,7 @@ public Integer getObject() {
 		createWorksitePanel.setVisible(false);
 				
 		confirmedFriendsButtonForm.add(createWorksitePanel);
-		
-		final AjaxButton createWorksiteButton = new AjaxButton("createWorksiteButton", confirmedFriendsButtonForm) {
-
-			private static final long serialVersionUID = 1L;
 
-			@Override
-			protected void onSubmit(AjaxRequestTarget target, Form<?> form) {
-				createWorksitePanel.setVisible(true);
-				target.add(createWorksitePanel);
-				target.appendJavaScript("fixWindowVertical();");
-			}
-
-		};
-		createWorksiteButton.setModel(new ResourceModel("link.worksite.create"));
-		createWorksiteButton.add(new AttributeModifier("title", true, new ResourceModel("link.title.worksite.create")));
```

Comment:
You need to remove the i18n string, too.
bundle/src/resources/ProfileApplication.properties

Make sure you remove it from all the translations.

Reply:
Thank you, @adrianfish;
You are right. 
Strings removed from i18n files.